Makefile

```diff
@@ -19,6 +19,9 @@ COMMIT_HASH ?= $(shell git rev-parse HEAD)
 # output in the $UI_SRC_ROOT/build/ subdirectory.
 CONSOLE_LOCAL_DIR ?= ${ROOTDIR}/../../../../../kiali-ui
 
+# The path where the version-compatibility-matrix.yaml has been git cloned.
+VERSION_METRIX_LOCAL_DIR ?= ${ROOTDIR}/version-compatibility-matrix.yaml
```

Comment:
I just realized in several places the word "matrix" is being misspelled - here (and elsewhere) you have "metrix" (with an "e" not an "a") --- the correct spelling is "matrix".

Reply:
Done.